implementations/rust/ockam/ockam_command/src/project/show.rs

```diff
@@ -1,6 +1,8 @@
 use clap::Args;
 use miette::IntoDiagnostic;
 
+use crate::terminal::tui::ShowCommandTui;
```

Comment:
You should remove the changes done in this file.

Reply:
Sorry again about carrying those change in all my PRs.